lxd/storage/backend_lxd_patches.go

```diff
@@ -124,3 +125,33 @@ func patchMissingSnapshotRecords(b *lxdBackend) error {
 
 	return nil
 }
+
+// patchCleanStorageVolumes deletes the remaining snapshot records in storage_volumes
+// (a previous patch would have already moved them into storage_volume_snapshots).
+func patchCleanStorageVolumes(b *lxdBackend) error {
```

Comment:
Lets change this function name to reflect the patch name. Can you also define "Remaining" in the description as to what it means and how we got into this situation.